io.github.hurricangame.hurrican.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.hurricangame.hurrican
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: hurrican
+rename-desktop-file: Hurrican.desktop
```

Comment:
you could just install it with the right name since the .deskop file is installed by the flatpak build

Reply:
True, but I plan to add the .desktop file to the game repo once I've coordinated with the other dev, that's why I did it this way as preparation.

Reply:
It should be submitted with the proper name anyway. See https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id

Reply:
Ok, renamed.